source/configuration_manual/imap_compress.rst

```diff
@@ -13,11 +13,32 @@ IMAP session.
 Settings
 ========
 
+.. versionremoved:: v2.4.0;v3.0.0
+
+  Configuration of the compression level and algorithm was dropped. The
+  extension is now enabled by default and configured with the default
+  compression level for the available mechanism.
+
 See :ref:`plugin-imap-zlib`.
 
 Configuration
 -------------
 
+To configure the compression level for the used mechanism use the following
+option:
+
+.. code-block:: none
+
+  # syntax: imap_compress_<algorithm>_level = value
```

Comment:
didn't we drop this?

Reply:
Yes. The first block in the settings section is that it was removed in v2.4.0/v3.0.0 There is unfortunately no obvious way to make the rest of the document structured below it without sacrificing readibility.

Reply:
We could just remove this block completely? It's still going to be available in 2.3 docs.